### DISCUSSION TITLE
Walmart Broken?

### DISCUSSION THREAD

**Discussion by theo-d-smith** [jef/streetmerchant#2783] [General]
Walmart must have changed their add to card button, or something, because it will say 'out of stock' for in stock items. Is anyone else having this problem? And is there anyone still active on this project that could fix something like this?

**Comment by Mr-Tech-13** [comment]
I'll take a peek when I get home from work

**Comment by Jwoods2019** [comment]
has anyone figured this out yet its definitely changed.

**Comment by Mr-Tech-13** [comment]
#__next > div:nth-child(1) > div > div > div > div > section > main > div > div:nth-child(2) > div > div.w_EC.w_FB.w_EG > div > div > div:nth-child(1) > div > div.pt3 > div > div:nth-child(3) > button (Walmart with text= Add to cart)
